swo_aws_extension/aws/client.py

```diff
@@ -243,3 +245,35 @@ def close_account(self, account_id):
                 logger.warning(f"Account {account_id} already closed")
             else:
                 raise
+
+    @wrap_boto3_error
+    def remove_account_from_organization(self, account_id):
+        """
+        Remove the account from the organization.
+
+        :return: None
+        """
+        org_client = self._get_organization_client()
+        try:
+            return org_client.remove_account_from_organization(AccountId=account_id)
+        except botocore.exceptions.ClientError as e:
+            if e.response["Error"]["Code"] == "ConstraintViolationException":
+                if "Try again later" in e.response["Error"]["Message"]:
+                    error = AWSTerminationCoolOffPeriodError(
+                        e.response["Error"]["Code"],
+                        e.response["Error"]["Message"],
+                        "Unlink Account",
+                        account_id,
+                    )
+                    raise error
```

Comment:
Better to log.info here, that we are waiting for cooloff period and we skip processing